### DISCUSSION TITLE
Can Children on path be broken apart?

### DISCUSSION THREAD

**Discussion by mcdonaldzachary** [microsoft/maker.js#607] [Q&A] [answered]
Hello when using children on path for example the text on arc demo. Can the path information of individual characters be accessed? I've tried my best to break them apart or access internal arrays but haven't had any luck. I'm specifically looking for location and bounding box 
Thanks 
https://maker.js.org/playground/?script=text-on-path

**Comment by danmarshall** [comment, marked as answer]
Yes, you can get characters of a Text model by their index. I've added this to the end of the function:

This gets the first character by `[0]` , gets its bounding box, then creates a rectangle of it:
```js
  const m = makerjs.measure.modelExtents(this.models.topText.models[0]);
  const bbox = new makerjs.models.Rectangle(m);
  this.models.bbox = bbox;
```

![image](https://github.com/user-attachments/assets/0ee69ee3-41b9-4fa3-8988-a4c0f8eb831f)

**Reply by mcdonaldzachary** [reply]
I would have never guessed that's how you get those arrayed items. Thank you so much I'll give that a try . I'm sure it'll work great

**Reply by danmarshall** [reply]
That's good feedback, there should be more documentation for the Text model.
